opencog/scm/CMakeLists.txt

```diff
@@ -6,6 +6,17 @@ INSTALL (FILES
 	DESTINATION "${DATADIR}/scm/"
 )
 
+# Create a symlink to load-path for site packages
+# NOTE: This is the first of the two symlinks required for this to work, as per
+# https://www.gnu.org/software/guile/manual/guile.html#Installing-Site-Packages
+INSTALL (CODE
+	"EXECUTE_PROCESS(
+		COMMAND ${CMAKE_COMMAND} -E make_directory
+		\"/usr/share/guile/site/2.0\"
```

Comment:
Also need to link to guile-2.2   I have not used 2.0 in years .. and 2.2 is a lot more stable for what I do (actually, 2.0 is unusable for what I do).